openassessment/fileupload/backends/base.py

```diff
@@ -0,0 +1,119 @@
+from django.conf import settings
+
+from ..exceptions import FileUploadInternalError
+from ..exceptions import FileUploadRequestError
+
+
+class Settings(object):
+    """Store settings related to file upload
+
+    The following settings are used:
+
+        FILE_UPLOAD_STORAGE_BUCKET_NAME (str, required): name of the bucket
+        (AWS or local file directory) to which content will be uploaded.
+
+        FILE_STORAGE_SETTING (str, defaults to "submissions_attachment"): this
+        will be used to prefix all stored file names. The specified file
+        prefix for the storage must be publicly viewable or all uploaded images
+        will not be seen.
+    """
+
+    @classmethod
+    def get_bucket_name(cls):
+        bucket_name = getattr(settings, "FILE_UPLOAD_STORAGE_BUCKET_NAME", None)
+        if not bucket_name:
+            raise FileUploadInternalError("No bucket name configured for FileUpload Service.")
+        return bucket_name
+
+    @classmethod
+    def get_prefix(cls):
+        return getattr(settings, "FILE_UPLOAD_STORAGE_PREFIX", "submissions_attachments")
```

Comment:
Nit: Perhaps this default value would be best pulled out into a separate variable with a comment.